core/src/main/java/greencity/exception/handler/HttpClientErrorExceptionResponse.java

```diff
@@ -3,8 +3,10 @@
 import com.fasterxml.jackson.annotation.JsonIgnore;
 import java.util.Map;
 import lombok.Value;
+import lombok.extern.slf4j.Slf4j;
 
 @Value
+@Slf4j
```

Comment:
never used

Reply:
Hi, thank you for catching those mistakes. For some reason, I can't reply to the above screenshot so I'll address it here. I'm refactoring this code now, though I have a question about test coverage. It's low because I'm adding and refactoring methods in classes that don't have any test classes, should I add those tests anyway?

Reply:
Yes, you have to add tests for all changes